src/test/fuzz/coins_view.cpp

```diff
@@ -74,6 +74,9 @@ FUZZ_TARGET_INIT(coins_view, initialize_coins_view)
             [&] {
                 (void)coins_view_cache.Flush();
             },
+            [&] {
+                (void)coins_view_cache.Sync();
+            },
```

Comment:
✅  This change addresses https://github.com/bitcoin/bitcoin/pull/17487#issuecomment-1403914399